tutorials/Qubit/Qubit.ipynb

```diff
@@ -163,7 +163,7 @@
     "> Before Q# 0.15 the syntax for qubit allocation was different:\n",
```

Comment:
Hmm, looking at this code block closer, it is actually a historical note on how the syntax looked like before the new `use` syntax, so it does make sense to use `using` here... Q# 0.15 was released exactly a year ago, so it probably makes sense to remove the historical part of the note, and switch it to describe the alternative current syntax
```suggestion
    "> An alternative syntax for qubit allocation defines the scope in which the allocated qubits are available explicitly:\n",
```

Reply:
And then we'll need to update line 170 as well, `'using' block` -> `'use' block` (GitHub doesn't let me leave a comment there directly)